CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Thank you for your interest in contributing! Join the [Plugin Marketplace channel](https://community.mattermost.com/core/channels/plugins-marketplace) on the Mattermost Community Server for discussion about the Plugin Marketplace.
```

Comment:
```suggestion
Thank you for your interest in contributing! Join the [Plugin Marketplace channel](https://community.mattermost.com/core/channels/plugins-marketplace) on the Mattermost Community server for discussion about the Plugin Marketplace.
```

Reply:
As pointed out in (https://github.com/mattermost/mattermost-marketplace/pull/45#discussion_r391944173) `Community Server` is used in other places.